### PR TITLE
pythonPackages.wordcloud: init at 1.6.0

### PR DESCRIPTION
**Pull request by jm2dev**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/wordcloud/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    rev = version;
+    sha256 = "1ncjr90m3w3b4zi23kw6ai11gxahdyah96x8jb2yn2x4573022x2";
+  };
+
+  propagatedBuildInputs = [ matplotlib numpy pillow ];
+
+  # Tests require extra dependencies
+  checkInputs = [ codecov coverage flake8 mock pytest pytestcov pytest-sugar setuptools twine wheel ];
+  checkPhase = ''
+    python setup.py test
```
Comment (jonringer) [inline pkgs/development/python-modules/wordcloud/default.nix]: this project isn't setup to have setuptools run the test suite, please change:
```suggestion
      # skip tests which make assumptions about installation
  checkPhase = ''
    pytest -k 'not cli_as_executable'
  '';
```

File: pkgs/development/python-modules/wordcloud/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  '';
+  
+  meta = with stdenv.lib; {
+    description = "A little word cloud generator in Python";
+    homepage = https://github.com/amueller/word_cloud;
```
Comment (jonringer) [inline pkgs/development/python-modules/wordcloud/default.nix]:
```suggestion
    homepage = "https://github.com/amueller/word_cloud";
```

File: pkgs/development/python-modules/wordcloud/default.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    repo = pname;
+    rev = version;
+    sha256 = "1ncjr90m3w3b4zi23kw6ai11gxahdyah96x8jb2yn2x4573022x2";
+  };
+
+  propagatedBuildInputs = [ matplotlib numpy pillow ];
+
+  # Tests require extra dependencies
+  checkInputs = [ codecov coverage flake8 mock pytest pytestcov pytest-sugar setuptools twine wheel ];
+  checkPhase = ''
```
Comment (jonringer) [inline pkgs/development/python-modules/wordcloud/default.nix]: please add comment as to why the tests are being skipped
```suggestion
  # skip tests which make assumptions about installation
  checkPhase = ''
```

Comment (jm2dev) [reply pkgs/development/python-modules/wordcloud/default.nix]: Sorry, my fault for missing the comment. Just added.

**Review by jonringer — APPROVED** [review]
LGTM
executable shows usage

```
https://github.com/NixOS/nixpkgs/pull/79388
2 package built:
python37Packages.wordcloud python38Packages.wordcloud
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build python37Packages.wordcloud python38Packages.wordcloud

**Pull request merged**